WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -167,6 +173,12 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.new_edit_post_activity);
 
+        // TODO: Remove debug build check when new editor is moved to production
+        if (PackageUtils.isDebugBuild()) {
```

Comment:
We should remove the `isDebugBuild` check here or hide the option in settings when not in a debug build. 

IMO, it's safe to remove it since we're not going to merge `feature/visual-editor` in `develop` until the editor is ready.